.scripts/upgrade-versions.mjs

```diff
@@ -21,6 +21,15 @@ for (const sample of STORED_SAMPLES) {
       }
     }
   }
+  for (const [dep, version] of Object.entries(content.devDependencies)) {
+    if (dep.startsWith('@temporalio/')) {
+      if (version !== versionToSet) {
+        content.devDependencies[dep] = versionToSet;
+        console.log(`Updated version for ${sample} devDependency ${dep} from ${version} to ${versionToSet}`);
+        changed = true;
+      }
+    }
```

Comment:
nit: move to function

Reply:
The internet has been telling me lately to stop drying every single thing, so I resisted the urge 😄 Will change